File: app/src/main/java/com/squizzard/MisriApplication.java
package com.squizzard;

import android.app.Application;

import com.facebook.stetho.Stetho;
import com.squizzard.broadcast.NotificationChannelHelper;

public class MisriApplication extends Application {

    private NotificationChannelHelper notificationChannelHelper = new NotificationChannelHelper();

    @Override
    public void onCreate() {
        super.onCreate();
        Stetho.initializeWithDefaults(this);
        notificationChannelHelper.createNotificationChannel(this);
    }
}

File: app/src/main/java/com/squizzard/MisriApplication.kt
package com.squizzard;

import android.app.Application;

import com.squizzard.broadcast.NotificationChannelHelper;

public class MisriApplication extends Application {

    private final NotificationChannelHelper notificationChannelHelper = new NotificationChannelHelper();

    @Override
    public void onCreate() {
        super.onCreate();
        notificationChannelHelper.createNotificationChannel(this);
    }
}
